### PR TITLE
Update Dashboard icons

### PR DESCRIPTION
**Pull request by SbenIrontec**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [ ] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Comment by Kaian** [comment]
- Assign the PR to Soraya
- Set milestone information based on PROVIDER-1644 fixed version
- Set labels *tempest*,*enhancement*,*portal/platform*,*portal/brand*,*portal/client*, *portal/user*
- Set reviewer based on Tech Reviewer on PROVIDER-1644

File: web/portal/client/src/components/Dashboard/cards/CardsAmountVpbx.tsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import _ from '@irontec/ivoz-ui/services/translations/translate';
 import { Link } from 'react-router-dom';
 
-import { DashboardData } from '../@types';
```
Comment (mmadariaga) [inline web/portal/client/src/components/Dashboard/cards/CardsAmountVpbx.tsx:4]: this is required

File: web/portal/client/src/components/Dashboard/cards/CardsAmountVpbx.tsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-import IconClientsDashboard from '../IconClients';
-import IconPlatformsDashboard from '../IconPlatforms';
-import IconUsersDashboard from '../IconUsers';
+import DialpadIcon from '../DialpadIcon';
```
Comment (mmadariaga) [inline web/portal/client/src/components/Dashboard/cards/CardsAmountVpbx.tsx:4]: There are actually four client portals, based on loged in client type (vpbx, retail, residential and wholesale). This change must be applied to CardsAmountResidential.tsx, CardsAmountRetail.tsx and CardsAmountWholeSale.tsx as well. 

I guess that IconClients, IconPlatforms and IconUsers are no longer needed so remove them.

File: web/portal/brand/src/components/Dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -20,9 +20,9 @@ import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { useStoreActions } from 'store';
 
-import IconClientsDashboard from './IconClients';
-import IconPlatformsDashboard from './IconPlatforms';
```
Comment (mmadariaga) [inline web/portal/brand/src/components/Dashboard/Dashboard.tsx:24]: Remove old icons

File: web/portal/platform/src/components/Dashboard/Dashboard.tsx [hunk 4]
```diff
@@ -23,9 +23,9 @@ import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { useStoreActions } from 'store';
 
-import IconClientsDashboard from './IconClients';
-import IconPlatformsDashboard from './IconPlatforms';
-import IconUsersDashboard from './IconUsers';
```
Comment (mmadariaga) [inline web/portal/platform/src/components/Dashboard/Dashboard.tsx:28]: Remove old clients

File: web/portal/user/src/components/Dashboard/Dashboard.tsx [hunk 5]
```diff
@@ -20,8 +20,9 @@ import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { useStoreActions } from 'store';
 
-import ChatBubbleIcon from './ChatBubbleIcon';
-import PhoneForwardedIcon from './PhoneForwardedIcon';
```
Comment (mmadariaga) [inline web/portal/user/src/components/Dashboard/Dashboard.tsx:24]: Remove old icons

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**